cpp/harvest/nacjd.cc

```diff
@@ -613,6 +643,71 @@ void getStatistics(int argc, char **argv) {
              + std::to_string(no_initial_date) + "\n" + "\t\tKeywords not found or empty: " + std::to_string(no_keywords) + "\n");
 }
 
+void getCategoriesStatistics(int argc, char **argv) {
```

Comment:
There is lots of copy/paste code here (sometimes more than necessary), and there is already a get_statistics mode in the usage. Please add the necessary logic to the existing mode instead of adding a new one.